daemon/graphdriver/vfs/driver.go

```diff
@@ -41,8 +48,10 @@ func Init(home string, options []string, uidMaps, gidMaps []idtools.IDMap) (grap
 // In order to support layering, files are copied from the parent layer into the new layer. There is no copy-on-write support.
 // Driver must be wrapped in NaiveDiffDriver to be used as a graphdriver.Driver
 type Driver struct {
-	home       string
-	idMappings *idtools.IDMappings
+	driverQuota
+	quotaSupported bool
```

Comment:
I think this can be done in a simple manner, without an extra `bool` field. The idea is to add `isQuotaSupported()` method. In `quota_linux.go`, it is just `return driver.quotaCtl != nil`, in `quota_unsupported.go`, return false straight away. To my mind, it will be cleaner/simpler.